FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/ApiService.kt

```diff
@@ -114,22 +114,15 @@ interface ApiService {
   @GET("https://{directHost}/.well-known/openpgpkey/policy")
   suspend fun checkPolicyForWkdDirect(@Path("directHost") directHost: String): Response<ResponseBody>
 
-  /**
-   * This method calls "https://flowcrypt.com/shared-tenant-fes/account/get"
-   */
-  @POST(BuildConfig.API_URL + "account/get")
-  suspend fun getClientConfigurationFromFlowCryptComBackend(
-    @Header("Authorization") authorization: String,
-    @Body body: Any = Any(),
-  ): Response<ClientConfigurationResponse>
-
   /**
    * This method calls "https://fes.$domain/api/v1/client-configuration?domain=$domain"
    *
-   * @param fesUrl URL of FES
+   * @param customFesUrl A custom FES URL
    */
   @GET
-  suspend fun getClientConfigurationFromFes(@Url fesUrl: String): Response<ClientConfigurationResponse>
+  suspend fun getClientConfigurationFromFes(
+    @Url customFesUrl: String? = BuildConfig.API_URL + "account/get"
```

Comment:
there isn't any `account/get` endpoint anymore, either. Client configuration comes from the FES client configuration endpoint

Reply:
Got it. Could you share `SHARED_TENANT_FES_URL`?

`apiService.getClientConfigurationFromFes(fesUrl = customFesUrl ?? SHARED_TENANT_FES_URL`

Reply:
That would be this. Let's rename it from `API_URL` to `SHARED_TENANT_FES_URL`

![image](https://user-images.githubusercontent.com/6306961/214104995-4233c7f0-f120-4caa-ab04-ac9b605f7019.png)